src/main/java/org/jenkinsci/plugins/github/pullrequest/builders/GitHubPRStatusBuilder.java

```diff
@@ -25,7 +28,9 @@
 /**
  * Sets pr status for build caused by GitHubPRCause
  */
-public class GitHubPRStatusBuilder extends Builder {
+public class GitHubPRStatusBuilder extends Builder implements SimpleBuildStep {
+    private static final long serialVersionUID = 1L;
```

Comment:
Should not be necessary. Build steps are not generally `Serializable`. (They may be persisted via XStream but that is not going to care about a `serialVersionUID`.)